firmware/Reload Pro.cydsn/ui.c

```diff
@@ -783,7 +868,7 @@ static state_func ui_calibrate(const void *arg) {
 	memcpy(&new_settings.calibration_settings, &default_settings.calibration_settings, sizeof(calibration_settings_t));
 	
 	Display_ClearAll();
-	Display_DrawText(0, 0, " CALIBRATION ", 1);
+	Display_DrawText(0, 0, " CALIBRATION", 1);

```

Comment:
The extra space here is to ensure that whatever was in that line previously gets erased.